tests/ui/pages/discopane.py

```diff
@@ -0,0 +1,40 @@
+from pypom import Page
+from selenium.webdriver.common.by import By
+
+
+class DiscoveryPane(Page):
+
+    _discopane_content_locator = (By.CSS_SELECTOR, '.disco-pane')
+    _play_video_locator = (By.CSS_SELECTOR, '.play-video')
+    _uninstalled_toggles_locator = (By.CSS_SELECTOR, '.switch.uninstalled')
+    _close_video_link_locator = (By.CSS_SELECTOR, '.close-video')
+    _see_more_addons_locator = (By.CSS_SELECTOR, '.amo-link')
+    _search_box_locator = (By.ID, 'lst-ib')
+
+    @property
+    def is_video_playing(self):
+        return self.is_element_present(*self._close_video_link_locator)
```

Comment:
We should use `is_element_displayed`, otherwise this might return true if the element is in the DOM but not displayed.